collections/chinese-trad-999-club.csv

```diff
@@ -0,0 +1,1001 @@
+name, token
+000, 零零零
+001, 零零壹
+002, 零零贰
+003, 零零叁

```

Comment:
adding on, numbers 2 and 6 as well. 
On the list: 贰 and 陆 respectively
Correct one: 貳 and 陸 respectively.

Reply:
In that case, I'd say renaming it would probably be better. However, I believe that this makes it far too confusing and it would simply be better to have a simplified and traditional CN 999 club. Makes it easy for people to understand.

Reply:
this is so confusing, leave the 999cn club only

Reply:
no need to had more chinese, there is one that is rasing now, no need to add one more and confuse everyone, leave the simpiflied.

Reply:
we most likely will only have one cn 999 club on site. this could change but as of right now 95% sure we are keeping one cn 999 club.

Reply:
too confusing for me, the 999 cn club is better, no need to have two

Reply:
No reason there shouldn't be a tradition CN 999 club, wouldn't confuse anyone as the current CN 999 club is noted as simplified.